third_party/2and3/boto/compat.pyi

```diff
@@ -1,10 +1,17 @@
+import sys
+
 from typing import Any
 from base64 import encodestring as encodebytes
 
 from six.moves import http_client
 
 expanduser = ...  # type: Any
-StandardError = ...  # type: Any
+
+if sys.version_info >= (3, 0):
+    StandardError = Exception
+else:
+    StandardError = __builtins__.StandardError
```

Comment:
It's actually called `__builtin__` in Python 2—not sure how this is working.

Reply:
I never completely understood this myself, but empirically, `__builtins__` is the name that `__builtin__` is implicitly imported as:

```
$ python
Python 2.7.14 |Anaconda, Inc.| (default, Mar 27 2018, 12:28:59) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> __builtins__
<module '__builtin__' (built-in)>
>>> import __builtin__ as builtin
>>> 
```

https://docs.python.org/2.7/reference/executionmodel.html says that users shouldn't touch `__builtins__` -- is it preferred that I `import __builtin__` depending on the python version?  Does that work cleanly in stubs?

```py
if sys.version_info >= (3, 0):
    StandardError = Exception
else:
    import __builtin__
    StandardError = __builtin__.StandardError
```

Reply:
Thanks for clarifying that. I think the current code is fine then.